pkg/render/tiers/tiers.go

```diff
@@ -121,7 +138,7 @@ func (t tiersComponent) allowTigeraClusterDNSPolicy() *v3.NetworkPolicy {
 					Action: v3.Allow,
 					Source: v3.EntityRule{
 						NamespaceSelector: "all()",
-						Selector:          DNSIngressNamespaceSelector,
+						Selector:          TigeraNamespaceSelector,
```

Comment:
I think it's strictly not needed, but I think it's OK for this policy to still exist when running with nodelocaldnscache

Reply:
sure - here we are just refactoring the selector to be used by both policies.
maybe later on when we have more confident in cluster running with nodelocaldnscache we can refactor a bit more here.

Reply:
sure - we just renamed the var to be used in both policy. Maybe later on when we have more confident in everything working with nodelocal dnscache, we can also delete this policy as part of watching for nodelocal dnscache in the cluster?